src/main.scss

```diff
@@ -1,11 +1,11 @@
-.section.tags,
```

Comment:
Removing the `.section.tags` and `.section.tags:hover` rules will break the theme for users not on DEV. So instead of replacing them with the new rule, just add the new rule alongside it.

Reply:
Nice catch, thanks! I'm still not used to this setup ^^'